nall/string/markup/node.hpp

```diff
@@ -75,8 +76,10 @@ struct Node {
       case Comparator::LE: if(numeral(data) <= numeral(side(1))) continue; break;
       case Comparator::GT: if(numeral(data)  > numeral(side(1))) continue; break;
       case Comparator::GE: if(numeral(data) >= numeral(side(1))) continue; break;
+      default: break;
       }
 
+      // NOT REACHED
```

Comment:
I did most of the reviews of this by comparing it to the bsnes PR. I must've read the diffs backwards.